src/app-directory/specification/appd.yaml

```
@@ -380,10 +365,36 @@ components:
           items:
             type: string
           description: >-
-            A comma sepaarted list of the types of contexts the intent offered by the application can process. 
+            A comma sepaarted list of the types of contexts the intent offered by the application can process.
```

```suggestion
            A comma separated list of context-types that the application's intent(s) can process.
```
Typo and rephrasing.